apps/docs/spec/supabase_dart_v2.yml

```diff
@@ -2512,84 +2527,85 @@ functions:
     examples:
       - id: with-select
         name: With `select()`
-        isSpotlight: true
         code: |
           ```dart
           final data = await supabase
-            .from('cities')
-            .select('name, country_id')
-            .ilike('name', '%la%');
-          ```
-      - id: with-update
-        name: With `update()`
-        code: |
-          ```dart
-          final data = await supabase
-            .from('cities')
-            .update({ 'name': 'Mordor' })
-            .ilike('name', '%la%');
-          ```
-      - id: with-delete
-        name: With `delete()`
-        code: |
-          ```dart
-          final data = await supabase
-            .from('cities')
-            .delete()
-            .ilike('name', '%la%');
-          ```
-      - id: with-rpc
-        name: With `rpc()`
-        code: |
-          ```dart
-          // Only valid if the Stored Procedure returns a table type.
-          final data = await supabase
-            .rpc('echo_all_cities')
-            .ilike('name', '%la%');
+            .from('countries')
+            .select()
+            .ilike('name', '%alba%');
           ```
+        data:
+          sql: |
+            ```sql
+            create table
+              countries (id int8 primary key, name text);
+
+            insert into
+              countries (id, name)
+            values
+              (1, 'Afghanistan'),
+              (2, 'Albania'),
+              (3, 'Algeria');
+            ```
+          response: |
+            ```json
+            {
+              "data": [
```

Comment:
the other sample responses directly return the `data` array without the wrapping object, which one is correct?

Reply:
Thanks for catching this one. Directly returning the array is the correct one.